src/lib/event_mapper/impl/step_functions_event_mapper.py

```diff
@@ -12,14 +12,18 @@ def get_resource_name(self, event):
         arn = event["detail"]["stateMachineArn"]
         return arn.split("stateMachine:")[1]
 
-    def get_service_name(self):
-        return SettingConfigResourceTypes.STEP_FUNCTIONS
+    def get_resource_state(self, event):
+        return event["detail"]["status"]
 
     def get_event_status(self, event):
         return event["detail"]["status"]
 
     def get_event_severity(self, event):  # todo: implement
-        return "Unknown"
+        return (
+            EventSeverity.ERROR
+            if self.get_resource_state(event) in ("FAILED", "TIMED_OUT")
```

Comment:
here I'd prefer to re-use failure stated definitions from step_functions_manager:

class StepFunctionsManager:
    STATES_SUCCESS = ["SUCCEEDED"]
    STATES_FAILURE = ["FAILED", "ABORTED", "TIMED_OUT"]
    
The  same is for glue jobs and glue_workflows (we also have such constants defined)

And, typically, there is a method there "is_final_state" - might be useful for us to decide if we want to put the event into cloudwatch log

Reply:
Changed to managers usage to determine the event result, however I'm not sure it's really beneficial to add another public method to event mapper to get the info if the state is final, while still provisioning the event result to mapper clients because we have a logic which filters out both info and success event results - for alerts. What do you think about keeping the filtration logic about the "monitorable" events a bit more flexible like it's implemented now and have it aligned with the "alertable" events filtration logic?